packages/scss/src/components/inputs/switch/_switch.scss

```
@@ -16,38 +15,44 @@
 }
 
 .switch-label {
-	color: _theme("commons.disabled.color");
 	cursor: pointer;
 	display: inline-block;
-	line-height: _component("switch.label.line-height");
-	padding: _component("switch.label.padding");
-	transition: color 150ms ease;
+	line-height: 1.5rem;
+	padding: 0 0 0 calc(2rem - 4px + #{_theme("spacings.smaller")});
```

```suggestion
	padding: 0 0 0 calc(2rem - 4px + #{_theme("spacings.smaller", true)});
```
you need to fetch the real value here before `calc` not point towards the css v ar